docker/.env.example

```diff
@@ -46,5 +56,5 @@ WIDGET_CONTEXT_PATH=
 
 # Analytics
 SENTRY_DSN=
-NEW_RELIC_APP_NAME=
-NEW_RELIC_LICENSE_KEY=
+NEW_RELIC_APP_NAME=new-relic-novu-app
+NEW_RELIC_LICENSE_KEY=new-relic-novu-license-key
```

Comment:
do we need the mock values here, I am afraid that if the user will not provide/provide invalid credentials it will be less discoverable this way.

Reply:
Do we have ref of the errors? because if it is a warning of not being able to connect to the service it should be fine and mock key won't solve it.

Additionally, I am not sure what behavior we will have with the mock of NEW_RELIC_LICENSE_KEY. I think the best one for us would be if `key` is provided but invalid do not initialize the app, if `key` is not provided skip metric initialization.